alf/utils/video_recorder.py

```diff
@@ -315,6 +364,42 @@ def _plot_value(self, value):
         steps."""
         return
 
+    def _plot_value_curve(self,
```

Comment:
This seems duplicate with the existing _plot_prob_curve (although it doesn't have to be probability)

Reply:
Merged